docs/language/overview.md

```diff
@@ -1474,7 +1450,7 @@ Searches for any string that begins with `foo` has the string
 
 Keywords and string literals are equivalent search terms so it is often
 easier to quote a string search term instead of using escapes in a keyword.
-Keywords are useful in interactive modes of use where searches can be issued
+Keywords are useful in interactive contexts where searches can be issued
```

Comment:
@nwt: Maybe I'm officially overthinking it at this point, but "settings" makes me think of configurable settings. I just pushed a change so that it now says "interactive workflows". I even found a spot earlier in the doc where text-not-written-by-me already said "interactive query workflows", so it even seems there's precedent.